api/base/serializers.py

```diff
@@ -1689,8 +1689,12 @@ def create(self, validated_data):
             raise api_exceptions.RelationshipPostMakesNoChanges
 
         for node in add:
-            collection.add_pointer(node, auth)
-
+            try:
+                collection.add_pointer(node, auth)
```

Comment:
Realized we're calling `add_pointer` other places and not catching potential errors.  There are other locations on this page (both creating and updating linked nodes/registrations) that need to catch the error, plus tests.